system_baseline/views/v1.py

```diff
@@ -701,13 +702,14 @@ def delete_systems_with_baseline(baseline_id, system_ids):
     return system_ids
 
 
-def create_deletion_request_for_systems(baseline_id, system_ids):
+def create_deletion_request_for_systems(baseline_id, body):
     ensure_rbac_write()
     validate_uuids([baseline_id])
+    system_ids = body["system_ids"]
     validate_uuids(system_ids)
 
-    return_value = delete_systems_with_baseline(baseline_id, system_ids)
-    return return_value
+    system_ids = delete_systems_with_baseline(baseline_id, system_ids)
+    return {"system_ids": system_ids}
```

Comment:
Actually, I approved, but when I was looking to see the best order to merge to prevent conflicts, I noticed this spot is shared in both and is not the same:
https://github.com/RedHatInsights/system-baseline-backend/pull/163/files#diff-285e0634d3b94393071de1f685907e13d80487fa2e250ae076868664473fbad3R709-R710